include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -441,6 +471,16 @@ __pattern_unique(_ExecutionPolicy&& __exec, _Range&& __rng, _BinaryPredicate __p
 // merge
 //------------------------------------------------------------------------
 
+template <typename _Name>
+class __copy_first_wrapper
+{
+};
+
+template <typename _Name>
+class __copy_second_wrapper
+{
+};
```

Comment:
Does it make sense to make naming more consistent? Here you use "first" and "second" for the names to differ, in other case you use "1" and "2"